engine-api/src/main/java/io/nosqlbench/engine/api/activityimpl/SimpleActivity.java

```diff
@@ -218,7 +220,7 @@ public void registerAutoCloseable(AutoCloseable closeable) {
     @Override
     public void closeAutoCloseables() {
         for (AutoCloseable closeable : closeables) {
-            logger.debug("CLOSING " + closeable.getClass().getCanonicalName() + ": " + closeable.toString());
+            logger.debug("CLOSING " + closeable.getClass().getCanonicalName() + ": " + closeable);
```

Comment:
Since errors on close() are generally non-recoverable, we can't _handle_ it better, but I did at least add the details and the exception to the logger call.